### DISCUSSION TITLE
[Question] Controlled traffic farming (CTF)

### DISCUSSION THREAD

**Discussion by Skog** [Courseplay/Courseplay_FS22#2853] [General]
Hi! I'm having problems with Controlled traffic farming (CTF) so I am doing 36M tramlines with every equipment being 12M. But when I want to do a course with 2 multiple tools, it doesn't match with the tramlines; when doing 3, it works again. When doing one tool with two headlands it works which is the same as one headland with two tools but that doesn't work. Everything works perfectly otherwise, so I'm just wondering why 2 aren't lining up.

**Comment by nitroSkirata** [comment]
It might be helpful if you added some illustrations to your post.

For rightnow I am not really sure what isn't lining up

**Comment by Tensuko** [comment]
I dont know what CTF is.
I only know proseed for this and even that we wont support.

**Comment by coatsy35** [comment]
Controlled traffic is the idea of only ever driving on the same tramlines year in year out, never moving from them. You have to use RTK signals in real life that never move, to do this. A lot of larger arable units in Europe are now on this system if they are direct drilling only or adopt a variation of it. You generally would then have implement widths that divide into the tramline widths i.e. 36m sprayer tramlines, would use a 12.3m header so you catch any overlap and then use a 6m or 12m cultivator and drill. When combining the unloader (usually a chaser bin but can be a trailer) would always sit on the tramline and never come off it. Meaning all heavy machines which cause the most damage (sprayer in wet periods and unloader) always stay on a tramline.

**Comment by coatsy35** [comment]
> Hi! I'm having problems with Controlled traffic farming (CTF) so I am doing 36M tramlines with every equipment being 12M. But when I want to do a course with 2 multiple tools, it doesn't match with the tramlines; when doing 3, it works again. When doing one tool with two headlands it works which is the same as one headland with two tools but that doesn't work. Everything works perfectly otherwise, so I'm just wondering why 2 aren't lining up.

You haven't got any overlap set have you? that would throw it out

**Comment by Tensuko** [comment]
Getting the exact same lines over and over again is nearly impossible. With headlands it is even harder.

**Comment by coatsy35** [comment]
Agreed.

**Comment by Skog** [comment]
> > Hi! I'm having problems with Controlled traffic farming (CTF) so I am doing 36M tramlines with every equipment being 12M. But when I want to do a course with 2 multiple tools, it doesn't match with the tramlines; when doing 3, it works again. When doing one tool with two headlands it works which is the same as one headland with two tools but that doesn't work. Everything works perfectly otherwise, so I'm just wondering why 2 aren't lining up.
> 
> You haven't got any overlap set have you? that would throw it out

Hi so no I have taken away the overlap, had that problem already when the 36M course and 12M course didn't line up=) But now it does

**Comment by Skog** [comment]
> Getting the exact same lines over and over again is nearly impossible. With headlands it is even harder.

Ye I know but there haven't been any problems before and I know that in fs19 it worked so I just took it for granted that it would work. I even think that I have done 12M with two tools before and it has line up to the 36M course.

**Comment by Skog** [comment]
![Fs22 12M x2](https://github.com/Courseplay/Courseplay_FS22/assets/47213175/8ab1c535-ea13-46f2-bf8d-4e78a39a1cef)
![Skärmbild fs 12M](https://github.com/Courseplay/Courseplay_FS22/assets/47213175/b367c33f-ff97-4788-a16b-4282e761f34d)

Here you can see what i mean with that the one 12 M equipment it llines up with the tramline but with two tool it doesn't

**Comment by nitroSkirata** [comment]
> ![Fs22 12M x2](https://user-images.githubusercontent.com/47213175/274739382-8ab1c535-ea13-46f2-bf8d-4e78a39a1cef.png) ![Skärmbild fs 12M](https://user-images.githubusercontent.com/47213175/274739409-b367c33f-ff97-4788-a16b-4282e761f34d.png)
> 
> Here you can see what i mean with that the one 12 M equipment it llines up with the tramline but with two tool it doesn't

Remember: in picture 1 one driver drives 6m to the left and one driver drives 6m to the right of the CP line.
therefore the line CAN'T be right on top of the tramline.

One of your drivers will therefore indeed run on the tramlines, even if it does not look that way in the course preview

**Comment by Skog** [comment]
![Tröska 12 x 2](https://github.com/Courseplay/Courseplay_FS22/assets/47213175/2da0e749-337f-4341-b0cc-e6ac8d52715d)
Ye I know but it's like 2 meters of witch is really odd

**Comment by nitroSkirata** [comment]
Can you post a screenshot of your generator settings?

**Comment by Tensuko** [comment]
Converted this to discussion.

**Comment by Skog** [comment]
> Can you post a screenshot of your generator settings?

![Skärmbild 2023-10-15 220822](https://github.com/Courseplay/Courseplay_FS22/assets/47213175/66354fda-800d-4d16-bf90-b7ca1e68d519)
![Skärmbild 2023-10-15 221228](https://github.com/Courseplay/Courseplay_FS22/assets/47213175/97edadbf-156b-4ab8-9105-6cd093b2429f)
Here are the setting for one tool only and when I want two tools I just change that and change to one headland
